deformable_manipulation_experiment_params/include/deformable_manipulation_experiment_params/task_enums.h

```diff
@@ -32,7 +33,7 @@ namespace smmap
     enum GripperControllerType
     {
         RANDOM_SAMPLING,
-        UNIFORM_SAMPLING
+        NOMAD_OPTIMIZATION
```

Comment:
Should we be adding the Gurobi based controller to this list as well?

Reply:
I think not? These enums are the way to solve the optimization problem for the new controller only, where the Gurobi doesn't work for the new controller; Probably rename the "GripperControllerType" to "NonlinearOptimizationType", or something else?

Reply:
We'll push this off to a later change, marked as #8 in the issue list.